### PR TITLE
DOCS-7893 SaaS Cost Integrations + Cloud Cost Management Doc Update

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do? What is the motivation?

Add a partial for Cloud Cost Management, add Data Collected sections in product landing page for shared + AWS/Google Cloud/Azure tags and SaaS Cost integration tags, add tabs for SaaS cost integrations in private beta on the in-app Accounts page.

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/cloud_cost_management/saas_costs.md [hunk 1]
```diff
@@ -27,90 +27,183 @@ further_reading:
...
+
+Your Fastly cost data for the past 15 months can be accessed in Cloud Cost Management after 24 hours.
+
+[101]: https://manage.fastly.com/account/personal/tokens
+[102]: https://app.datadoghq.com/integrations/fastly
+
+{{% /tab %}}
+{{% tab "Twilio" %}}
+
+<div class="alert alert-warning">The Twilio SaaS Cost integration is in private beta.</div>
```
Comment (alai97) [inline content/en/cloud_cost_management/saas_costs.md:188]:
```suggestion
```

**Review by aliciascott — APPROVED** [review]
Hey @alai97 this looks great! Quick question the Confluent Cloud, Snowflake , and Fastly tabs were already available I see, do they need a "public beta" callout to help differentiate those ? Either way is fine just wasn't sure!

File: content/en/cloud_cost_management/saas_costs.md [hunk 2]
```diff
@@ -27,90 +27,344 @@ further_reading:
...
+SaaS Cost Integrations allow you to send cost data **directly from your providers** by configuring the accounts associated with your cloud cost data in Datadog. 
 
-## Configure SaaS Cost Integrations
+If your provider is not supported, use [Custom Costs][1] to upload any cost data source to Datadog and understand the total cost of your services.
 
-To use SaaS Cost Integrations, you must [configure Cloud Cost Management][1] for either AWS, Azure, or Google Cloud.
+## Setup
 
-Navigate to [**Infrastructure > Cloud Costs > Settings > Accounts**][8] and click on the provider you want to integrate. 
+To use SaaS Cost Integrations, you must configure [Cloud Cost Management][2] for AWS, Azure, or Google Cloud.
```
Comment (alai97) [inline content/en/cloud_cost_management/saas_costs.md:41]:
```suggestion
To use SaaS Cost Integrations, you must configure [Cloud Cost Management][2] for AWS, Azure, or Google Cloud. 
See the respective documentation for your cloud provider:
```

File: content/en/cloud_cost_management/_index.md [hunk 3]
```diff
@@ -46,6 +46,163 @@ Datadog ingests your cloud cost data and transforms it into queryable metrics. I
   {{< nextlink href="/cloud_cost_management/saas_costs">}}<u>SaaS Cost Integrations</u>: Send cost data from a supported SaaS cost provider to Datadog. {{< /nextlink >}}
  {{< /whatsnext >}}
 
+## Data Collected
+
+Cloud Cost Management collects data for the AWS, Azure, and Google Cloud integrations, as well as shared data. The table provides insights into the specific tags associated with each integration.
```
Comment (alai97) [inline content/en/cloud_cost_management/_index.md:51]:
```suggestion
Cloud Cost Management collects data for the AWS, Azure, and Google Cloud integrations, as well as shared data. 

This table provides a non-exhaustive list of out-of-the-box tags associated with each integration.
```

Comment (crousseaux) [reply content/en/cloud_cost_management/_index.md:51]:
```suggestion
Cloud Cost Management collects tags for the AWS, Azure, and Google Cloud integrations. This table provides a non-exhaustive list of out-of-the-box tags associated with each integration.
```

**Review by goeln — APPROVED** [review]

File: content/en/cloud_cost_management/saas_costs.md [hunk 4]
```diff
@@ -27,105 +27,296 @@ further_reading:
...
+{{< img src="cloud_cost/saas_costs/fastly_setup.png" alt="Integrate with Fastly to collect cost data." style="width:100%" >}}
+
+[101]: https://manage.fastly.com/account/personal/tokens
+[102]: https://app.datadoghq.com/integrations/fastly
+
+{{% /tab %}}
+{{% tab "Twilio" %}}
 
-For more information, see the [Custom Costs documentation][9].
+<div class="alert alert-warning">The Twilio SaaS Cost integration is in private beta.</div>
```
Comment (alai97) [inline content/en/cloud_cost_management/saas_costs.md:196]:
```suggestion
<div class="alert alert-warning">The Twilio SaaS Cost integration is in public beta.</div>
```

**Pull request merged**